index.js

```diff
@@ -14,6 +14,10 @@ function defaultKey (req, file, cb) {
   })
 }
 
+function defaultAcl (req, file, cb) {
+  setImmediate(function () { cb(null, 'private') })
```

Comment:
Could you make this `setImmediate(cb, null, 'private')` to avoid creating an anonymous function.